src/main/java/MicroRaiden.java

```diff
@@ -85,8 +212,529 @@ public void listBalances(String account, String remotePeer) {
+    	ECKey keyPair=getECKeyByName(accountName);
+    	if(keyPair==null) {
+    		System.out.println("Cannot load account with name "+accountName+".");
+    		return;
+    	}
+        
+        String address = "0x"+new String(Hex.encodeHex(keyPair.getAddress()));
+        
+        CallTransaction.Function balanceOf = tokenContract.getByName("balanceOf");
+        byte [] functionBytes=balanceOf.encode(address);
+        String requestString = "{\"method\":\"eth_call\"," +
+                "\"params\":[" +
+                "{" +
+                "\"to\":\""+tokenAddr+"\"," +
+                "\"data\":\""+"0x" + new String(org.apache.commons.codec.binary.Hex.encodeHex(functionBytes))+"\"" +
+                "}," +
+                "\"latest\"" +
+                "]," +
+                "\"id\":42,\"jsonrpc\":\"2.0\"}";
+        if(debugInfo) {
+        	System.out.println("Request in getTokenBalance = "+requestString);
+        }
+        String myTokenBalance="";
+        try {
+        	myTokenBalance=getHttpResponse(requestString);
+        }catch (IOException e) {
+        	System.out.println("Cannot get token balance for "+accountName);
+    		return;
+        }
+    	System.out.println("Balance = "+new Float(new BigInteger(myTokenBalance.substring(2),16).floatValue()/(new BigInteger(appendingZerosForETH,10).floatValue())).toString()+" TKN");
+    	
+    }
+    
+    private static ECKey getECKeyByName(String accountName) {
+    	JSONParser parser = new JSONParser();
+    	JSONObject jobj=new JSONObject();
+        try {     
+        	jobj = (JSONObject)parser.parse(new FileReader(accountName+".pkey"));
+            
+        } catch (FileNotFoundException e) {
+        	System.out.println("Couldn't locate account file " + accountName + ".pkey");
+        	return null;
+        } catch (ParseException e) {
+        	System.out.println("Couldn't parse contents in " + accountName + ".pkey as a JSON object.");
+        	return null;
+        } catch (IOException e) {
+        	System.out.println("Couldn't parse contents in " + accountName + ".pkey as a JSON object.");
+        	return null;
+        }
+        String  privateKeyHex = (String) jobj.get("privkey");
+        try{
+            return ECKey.fromPrivate(Hex.decodeHex(privateKeyHex.toCharArray()));
+        }catch (DecoderException e) {
+        	System.out.println("PrivateKeyHex = " + privateKeyHex + " is invalid.");
+        	return null;
+        } 
+    }
+	private static byte[] bigIntegerToBytes(BigInteger value) {
+        if (value == null)
+            return null;
+
+        byte[] data = value.toByteArray();
+
+        if (data.length != 1 && data[0] == 0) {
+            byte[] tmp = new byte[data.length - 1];
+            System.arraycopy(data, 1, tmp, 0, tmp.length);
+            data = tmp;
+        }
+        return data;
+    }
+    
+    public void createChannel(String senderAccountName, String receiverAccountID, String deposit) {
```

Comment:
prob would be better to have the deposit be an int / float or whatever and then do a proper conversion from the arg handling function. In the longer run when people will be using this function in their own code I doubt they'll want to deal with token values as strings.